packages/web-components/fast-components/src/flipper/flipper.definition.ts

```diff
@@ -29,13 +28,13 @@ export const fastFlipperDefinition: WebComponentDefinition = {
                     description: "The direction attribute",
                     values: [
                         {
-                            name: FlipperDirection.previous,
+                            name: "previous",
```

Comment:
For some reason I can't resolve enums coming from `fast-foundation`. If someone has a clue as to why that might be happening let me know, as of right now these all had to change to strings.

Reply:
It's possible these are const enums, which could make TS remove them by default. There's a compiler setting to have it generate code for const enums if that's the case.

Reply:
Should they be changed to `type` as was done for a few other components recently? So `type FlipperDirection = "previous" | "next";`? Tagging @chrisdholt as I believe he made the change to other components. Also if we decide to do that I can leave these as strings, if we export them as enums I worry we will face a breaking change in the future if we want to backtrack and change the `enum` to `type` so maybe we should confirm that now.

Reply:
Maybe you just needed to re-build? I don't get an error importing and using this enum; it is not a const enum and is exported.

Reply:
You were right, just needed a rebuild, thanks!